Library/Formula/irssi.rb

```diff
@@ -21,10 +27,19 @@ class Irssi < Formula
   depends_on "openssl" => :recommended
   depends_on "dante" => :optional
 
-  # Fix Perl build flags and paths in man page
+  # Make paths in man page Homebrew-specific
   patch :DATA
 
   def install
+    # Fix Perl build flags
+    conf = build.head? ? "configure.ac" : "configure"
+    inreplace conf do |s|
```

Comment:
So it's trying to build universally unconditionally? I think `superenv` will actually handle this for us now. Mind checking that out, @DomT4 (i.e. seeing if you remove the patch if it makes a universal binary)?